tools/convert_datasets/dancetrack/dancetrack2coco.py

```diff
@@ -0,0 +1,165 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+# This script converts DanceTrack labels into COCO style.
+# Official repo of the DanceTrack dataset:
+# https://github.com/DanceTrack/DanceTrack
+#
+# Label format of DanceTrack dataset:
+#   GTs:
+#       <frame_id> # starts from 1 but COCO style starts from 0,
+#       <instance_id>, <x1>, <y1>, <w>, <h>,
+#       <conf> # conf is annotated as 0 if the object is ignored,
+#       <class_id>, <visibility>
+#
+#   DETs and Results:
+#       <frame_id>, <instance_id>, <x1>, <y1>, <w>, <h>, <conf>,
+#       <x>, <y>, <z> # for 3D objects
+#
+# Classes in DanceTrack:
+#   1: 'pedestrian'
+#
+#   USELESS classes are not included into the json file.
+#   IGNORES classes are included with `ignore=True`.
+#
+# This file is adapted from the data label conversion file for MOT
+# But as Dancetrack does not provide public detections and provides
+# official train/val/test splitting, we make necessary adaptation.
+
+import argparse
+import os
+import os.path as osp
+from collections import defaultdict
+
+import mmcv
+from tqdm import tqdm
+
+USELESS = [3, 4, 5, 6, 9, 10, 11]
+IGNORES = [2, 7, 8, 12, 13]
+
+
+def parse_args():
+    parser = argparse.ArgumentParser(
+        description='Convert DanceTrack label and detections to \
+        COCO-VID format.')
+    parser.add_argument('-i', '--input', help='path of MOT data')
+    parser.add_argument(
+        '-o', '--output', help='path to save coco formatted label file')
+    return parser.parse_args()
+
+
+def parse_gts(gts, is_mot15):
```

Comment:
remove `is_mot15` for dancetrack dataset

Reply:
Done.